src/__tests__/test.js

```diff
@@ -1,11 +1,12 @@
 /* @flow */
+import { describe, it } from "vitest";
 
 import {
```

Comment:
working on this file now so CI will fail. Just noticed it after hastily opening PR

Reply:
done